pkg/validate/runtime_info.go

```diff
@@ -21,32 +21,24 @@ import (
 	internalapi "k8s.io/kubernetes/pkg/kubelet/api"
```

Comment:
I think it's better to also move `TestGetRuntimeStatus ` and `TestGetVersion` into `runtime_info.go`